### DISCUSSION TITLE
v5.8 Tutorial registerDirs v.s. setDirectories

### DISCUSSION THREAD

**Discussion by milacic** [phalcon/cphalcon#16628] [General]
I was trying to update my app from v4 to v5.8 

Therefore I think I found mistate in tutorial for v5.8, probably in older once too.

https://docs.phalcon.io/5.8/application/

registerDirs should be replaced with setDirectories ... or maybe I'm wrong

**Comment by niden** [comment]
@milacic you are not wrong. I have corrected this. Thank you.
